InformationScripting/src/parsing/SimpleQueryParser.cpp

```diff
@@ -71,14 +71,33 @@ QueryNode* SimpleQueryParser::parseAny(const QString& queryString, int& index)
 		if (index < queryString.length())
 			ch = queryString[index];
 	}
-	else query = new CommandNode(commandString); // Create the command. Note that it could be an empty string.
+	else
+	{
+		// Create the command. Note that it could be an empty string.
+		auto parts = commandString.split(" ");
```

Comment:
Seems to work just fine. Except spaces in arguments are not allowed, but I guess that is fine.

Styling would be nice.